tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml

```diff
@@ -12,3 +27,150 @@ jobs:
+  pool: 'Win-CPU'
+  variables:
+    buildDirectory: '$(Build.BinariesDirectory)'
+    buildConfig: 'RelWithDebInfo'
+    buildArch: 'x86'
+
+  steps:
+    - template: templates/set-test-data-variables-step.yml
+
+    - template: templates/windows-build-tools-setup-steps.yml
+      parameters:
+        EnvSetupScript: 'setup_env.bat'
+        buildArch: $(buildArch)
+        setVcVars: false
+
+    - template: templates/windows-build-and-test-steps.yml
+      parameters:
+        buildAdditionalParams: ' --use_openmp --x86 '
+        buildArch: $(buildArch)
+        msbuildPlatform: 'Win32'
+        buildConfig: $(buildConfig)
+
+    - template: templates/c-api-artifacts-package-and-publish-steps-windows.yml
+      parameters:
+        buildConfig: $(buildConfig)
+        artifactName: 'onnxruntime-win-$(buildArch)'
+
+    - template: templates/clean-agent-build-directory-step.yml
+
+
+- job: Windows_Packaging_GPU_x64
+  pool: 'Win-GPU'
+  variables:
+    buildDirectory: '$(Build.BinariesDirectory)'
+    buildConfig: 'RelWithDebInfo'
+    buildArch: 'x64'
+
+  steps:
+    - template: templates/set-test-data-variables-step.yml
+
+    - task: CmdLine@2
+      displayName: 'Set CUDA 9.1 path'
+      inputs:
+        script: |
+          set PATH=C:\local\cuda-9.1.85-windows10-x64-0\bin;C:\local\cudnn-9.1-windows10-x64-v7.1\cuda\bin;%PATH%
+        modifyEnvironment: true
+        workingDirectory: '$(Build.BinariesDirectory)'
+
+    - task: PowerShell@2
+      displayName: 'Set CUDA 9.1 MSBuild properties'
+      inputs:
+        targetType: 'filePath'
+        filePath: '$(Build.SourcesDirectory)/tools/ci_build/github/windows/set_cuda_path.ps1'
+        arguments: '-CudaMsbuildPath C:\local\cudaMsbuildIntegration-9.1.85-windows10-x64-0 -CudaVersion 9.1'
+
+    - template: templates/windows-build-tools-setup-steps.yml
+      parameters:
+        EnvSetupScript: 'setup_env.bat'
+        buildArch: 'amd64'  # amd64 is needed for vcvars target arch
+        setVcvars: true
+
+    - task: CmdLine@2
+      displayName: 'Build and Test OnnxRuntime'
+      inputs:
+        script: |
+          $(Build.BinariesDirectory)\packages\python\python.exe $(Build.SourcesDirectory)\tools\ci_build\build.py --config $(buildConfig) --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --build_shared_lib --cmake_path $(Build.BinariesDirectory)\cmake\bin\cmake.exe --ctest_path $(Build.BinariesDirectory)\cmake\bin\ctest.exe --enable_onnx_tests --test_data_url $(TestDataUrl) --test_data_checksum $(TestDataChecksum) --use_openmp --msvc_toolset=14.11 --use_cuda --cuda_version 9.1 --cuda_home="C:\local\cuda-9.1.85-windows10-x64-0" --cudnn_home="C:\local\cudnn-9.1-windows10-x64-v7.1\cuda" 
+        workingDirectory: '$(Build.BinariesDirectory)'
+
+    - template: templates/c-api-artifacts-package-and-publish-steps-windows.yml
+      parameters:
+        buildConfig: $(buildConfig)
+        artifactName: 'onnxruntime-win-gpu-$(buildArch)'
+
+    - task: PowerShell@2
```

Comment:
Could you please remove this task?

Reply:
Will fix it later. This PR is to fix an urgent issue blocking release.
Thanks!

Reply:
please create a task to track this.